bundles/ranvier-areas/areas/limbo/rooms.yml

```diff
@@ -1,19 +1,19 @@
 - id: 1
```

Comment:
I would consider changing these room IDs as well, and if you want to be creative, titling them something other than "Test Room 1" and "Test Room 2". Thoughts?